src/components/LHNOptionsList/styles.native.js

```diff
@@ -0,0 +1,10 @@
+import styles from '../../styles/styles';
+
```

Comment:
This isn't self-explanatory, could you please add a comment explaining why we need different styles for different platforms? And a link to the GH issue